src/pages/index.js

```diff
@@ -122,7 +124,7 @@ function Home() {
         )}
         <div className={clsx("hero hero--secondary", styles.heroBanner)}>
           <div className="container">
-            <h2 className="hero__title">Want to know what's new?</h2>
+            <h2 className="hero__title">What's new</h2>
```

Comment:
💭 We should have some guidance about how/when the content in this section gets updated.

Reply:
Great idea 👍

Reply:
Perhaps by each alpha or minor release?